tests/e2e/conftest.py

```
@@ -0,0 +1,272 @@
+This module provides shared fixtures and utilities for end-to-end testing
+of the LedFx web interface using Playwright.
+"""
+
+import glob
+import os
+import subprocess
+import time
+from collections.abc import Generator
+
+import pytest
+from PIL import Image
```

Flake8 (7.3.0)

[error] 'PIL.Image' imported but unused

(F401)